pkg/archive/archive_unix_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	"testing"
```

Comment:
Any reason [`syscall.Stat_t`](https://golang.org/pkg/syscall/#Stat_t) is still being used in this file (instead of [`unix.Stat_t`](https://godoc.org/golang.org/x/sys/unix#Stat_t))?